creditnote.go

```diff
@@ -53,6 +62,27 @@ type CreditNoteListParams struct {
 	Invoice    *string `form:"invoice"`
 }
 
+// CreditNoteLineItemListParams is the set of parameters that can be used when listing credit note line items.
+type CreditNoteLineItemListParams struct {
+	ListParams `form:"*"`
+
+	// ID is the credit note ID to list line items for.
+	ID *string `form:"-"` // Goes in the URL
```

Comment:
Hmm. I would've expected this to be called `CreditNote` ? 

Or like CreditNoteID or something?

Reply:
Yeah the name is a bit random, it depends what the team puts in their url path. In a way it doesn't fully matter. I'm happy to go with `CreditNote` instead but usually we put `ID` unless a resource has multiple parent types (rare).

Reply:
ack